.npmignore

```diff
@@ -1,10 +1,4 @@
-.DS_Store
-node_modules
-npm-debug.log
-*.orig
-*.log
-tmp
-test
-
-# ignore emacs scratch files
-*~
+!dist/
+!dist-test/
+!docs/
+!es5/
```

Comment:
You might want to pick up the latest changes in the generator - the `.npmignore` is empty and what gets included in the package is explicitly outlined in `package.json`'s `"files"` field.